skynet/modules/stt/streaming_whisper/state.py

```diff
@@ -57,65 +56,86 @@ def _extract_transcriptions(
         final = ''
         interim = ''
         # return a final and flush the working audio if too many silent chunks
-        # or if the buffer is over the max length threshold
+        # or if the working audio is over the max duration threshold
         if (
             self.silent_chunks >= self.final_after_x_silent_chunks
             or utils.convert_bytes_to_seconds(self.working_audio) > self.force_final_duration_threshold
         ):
             if ts_result.text.strip() != '':
-                results.append(self.get_response_payload(ts_result.text, True))
-                self.all_final += ts_result.text
-                self.reset(True)
+                start_timestamp = int(ts_result.words[0].start * 1000) + self.working_audio_starts_at
```

Comment:
What does start contain?

Reply:
the floating seconds of when the word starts since the beginning of the working audio buffer